tests/test_root_entrypoint.py

```diff
@@ -0,0 +1,3 @@
+def test_root_entrypoint(client):
```

Comment:
why do we need this test?

Reply:
It's testing the communication between the user and the server,
And it's a required test as we learned in class.